Source/Core/DolphinQt/Settings/AudioPane.h

```diff
@@ -62,4 +65,7 @@ class AudioPane final : public QWidget
   QLabel* m_stretching_buffer_label;
   QSlider* m_stretching_buffer_slider;
   QLabel* m_stretching_buffer_indicator;
+
+  QString GetDPL2QualityLabel(int value);
```

Comment:
These should be specified with the rest of the function definitions above this. They can also be const member functions.